roxie/ccd/ccdserver.cpp

```diff
@@ -9237,7 +9301,10 @@ class CRoxieServerThroughSpillActivity : public CRoxieServerActivity
         startError.clear();
         readError.clear();
         if (state != STATEreset) // make sure input is only reset once
+        {
             CRoxieServerActivity::reset();
+            simpleSinkCalculated = false;  // Can this vary between child query calls?
```

Comment:
If this is true then the list of sinks is going to keep growing - I'm not sure that happens (I hope not). In that case this isn't needed.